### DISCUSSION TITLE
Sharing zkanji

### DISCUSSION THREAD

**Discussion by MarcusAseth** [z1dev/zkanji#44] [Q&A]
Someone learned of zkanji from me and they wanted to try it, so to avoid them that bug I've found in the beta version and to spare them downloading VC and Qt to get the latest version, I've packed my Release compiled version and uploaded on google drive for them at this link -> https://drive.google.com/file/d/1LIuFZQ5Sd0KlycYd9CshQW6bT7ldQR-d/view?usp=sharing
I've also added my .ini so that they get my "dark mode" color settings and a readme about that, is it ok for me to do so or do you prefer if I wouldn't? ^_^'

**Comment by z1dev** [comment]
I accidentally answered in mail since now I get notification :D I'll paste it here too:
> It's fine by me! zkanji is FOSS, and in general you can share programs like this, if you provide a link to the source. I don't care about the formalities that much though. Besides it's mentioned in the about-box.
>
> There is a legal issue that could come up with the dictionary, since it's from a different project. You don't have to ask for their permission > either, but it's safest if you include all the legal txt and readme files, so they can't complain.

**Reply by MarcusAseth** [reply]
Thanks :D


> I accidentally answered in mail since now I get notification :D

the only email I've received today is the Notification that a reply was added to this topic - which means your current mail provider is not only not getting notifications from github, but also not sending to gmail
raises the question about how many more emails you sent that where never received x_x   (sorry for bringing this bad news)

Edit: forget what I said, maybe the mail you answered to was the notification mail and not my email :)

**Reply by z1dev** [reply]
Yeah you are right, I need to pay more attention next time. Since it's been a while I got notification, I didn't realize they don't give me a real return address. The mail was sent successfully, which isn't surprising since it's gmail!
